packages/combo-box/test/toggling-dropdown.test.js

```diff
@@ -317,36 +318,6 @@ describe('toggling dropdown', () => {
     });
   });
 
-  describe('disabled', () => {
-    beforeEach(() => {
-      comboBox.disabled = true;
-    });
-
-    it('toggleIcon should not be hidden when disabled', () => {
-      expect(getComputedStyle(comboBox._toggleElement).display).not.to.equal('none');
-    });
-
-    it('dropdown should not be shown when disabled', () => {
-      click(input);
-      expect(comboBox.opened).to.be.false;
-    });
-  });
-
-  describe('read-only', () => {
-    beforeEach(() => {
-      comboBox.readonly = true;
-    });
-
-    it('toggleIcon should not be hidden when read-only', () => {
-      expect(getComputedStyle(comboBox._toggleElement).display).not.to.equal('none');
-    });
-
-    it('dropdown should not be shown when read-only', () => {
```

Comment:
The appearance of the `readoly` combo-box toggle button is also covered by visual tests.

Reply:
The test makes sure `!this.readonly` is checked [here](https://github.com/vaadin/web-components/blob/59b9864003c9dec13d50c7687d48cde43daa32bd/packages/combo-box/src/vaadin-combo-box-mixin.js#L385)

If I remove the check, a read-only combo-box can be opened, but none of the unit tests fail.

https://user-images.githubusercontent.com/1222264/184373625-d6898ce1-f803-4ec6-a658-8a447e67afe0.mp4

Reply:
My bad, copied the test for `disabled`. Fixed.